src/test/rpc/ValidatorRPC_test.cpp

```diff
@@ -385,12 +385,23 @@ class ValidatorRPC_test : public beast::unit_test::suite
         }
     }
 
+    void
+    test_validation_create()
+    {
+        using namespace test::jtx;
+        Env env{*this};
+        auto result = env.rpc("validation_create");
+        BEAST_EXPECT(result.isMember(jss::result) &&
+                     result[jss::result][jss::status] == "success");
```

Comment:
I'd be in favor of getting more coverage with an additional test with the optional seed parameter (which @ximinez may have originally suggested) even though that case worked before and after this fix.

Reply:
I'll add that test.